python/ray/_private/services.py

```diff
@@ -1359,22 +1387,22 @@ def start_gcs_server(redis_address,
     Returns:
         ProcessInfo for the process that was started.
     """
-    gcs_ip_address, gcs_port = redis_address.split(":")
-    redis_password = redis_password or ""
-    config_str = serialize_config(config)
-    if gcs_server_port is None:
-        gcs_server_port = 0
+    assert gcs_server_port > 0
```

Comment:
`if gcs_server_port <= 0: raise ValueError(...)`

Reply:
This is an internal assertion, i.e. `gcs_server_port` should have a non-zero value. Otherwise this is a coding error. No user should see this message.

Reply:
I'm not sure about this. I checked the code and I found no place it's checked before this function. So the user will see this I think.
Let me know if I'm wrong about this and I'll be ok to let it go.